### ISSUE TITLE
Training step

### ISSUE THREAD

**Issue by bobp26** [lxtGH/DecoupleSegNets#21]

**Comment by bobp26** [comment]
goog job，wish you could  fixed my doubts。Are the parameters of your second-stage training completely written in accordance with the sh script file for example：train_cityscapes_ResNet50_deeplab_decouple.sh set the learning rate set to 0.005，is it too high? are There  limit to the optimization category or the number of boundary relaxation epochs？
